tests/unit/components/Navigator.spec.js

```diff
@@ -248,6 +251,42 @@ describe('Navigator', () => {
     expect(errorSpy).toHaveBeenCalledWith(`Reference for "${identifier}" is missing`);
   });
 
+  it('correctly computes `isSpecificOverload` if hash does not exist', () => {
+    // non-symbol pages don't have valid hash
+    const wrapper = createWrapper({
+      propsData: {
+        symbolKind: undefined,
+      },
+    });
+    expect(wrapper.find(NavigatorCard).props('isSpecificOverload')).toBe(false);
+
+    // symbol page with no hash
+    wrapper.setProps({ symbolKind: 'method' });
+    expect(wrapper.find(NavigatorCard).props('isSpecificOverload')).toBe(false);
+  });
+
+  it('correctly identifies whether a hash is valid', () => {
+    // symbol with valid hash
+    let wrapper = createWrapper({
+      mocks: {
+        $route: {
+          path: '/documentation/Foo-abc12',
+        },
+      },
+    });
+    expect(wrapper.find(NavigatorCard).props('isSpecificOverload')).toBe(true);
```

Comment:
add a case where the path has no dash or has more than 1 dash.

Reply:
The default is no dash. I will add a multiple dash case.

Reply:
The test above this tests for no dash/hash. I will add a case for more than 1